src/mesh/voxel_cloud.rs

```diff
@@ -1,9 +1,9 @@
-use std::f32;
 use std::iter;
+use std::{f32, i32};
```

Comment:
You can remove the `i32` import, because we are now using `i32::max_value()` instead of `i32::MAX`. Don't know why the compiler doesn't warn us here.